networkx/algorithms/centrality/tests/test_harmonic_centrality.py

```diff
@@ -92,3 +94,21 @@ def test_singleton(self):
         c = harmonic_centrality(G, distance="weight")
         d = {0: 0}
         assert c == d
+
+    def test_cycle_c4_directed(self):
+        c = harmonic_centrality(self.C4_directed, nbunch=[0, 1], sources=[1, 2])
+        d = {0: 0.833, 1: 0.333}
+        for n in sorted([0, 1]):
+            assert almost_equal(c[n], d[n], places=3)
+
+    def test_p3_harmonic_subset(self):
+        c = harmonic_centrality(self.P3, sources=[0, 1])
+        d = {0: 1, 1: 1, 2: 1.5}
+        for n in sorted(self.P3):
```

Comment:
I don't think the `sorted` would be necessary here either, unless I'm misunderstanding something!